src/common/localization/es-cl.ts

```diff
@@ -161,11 +163,43 @@ export const spanish: IAppStrings = {
             tfRecords: "Registros de Tensorflow",
             tfPascalVoc: "Tensorflow Pascal VOC",
         },
+        messages: {
+            saveSuccess: "Configuración de exportación guardada correctamente",
```

Comment:
Se guardó correctamente la configuración de exportación

Reply:
We should do a per language pass at some point - I've only been using bing translator.